### PR TITLE
Theme tasks

### PR DESCRIPTION
**Pull request by dkorhov**

### CODE REVIEW

File: docroot/sites/all/modules/custom/theme_tasks/theme_tasks.info [hunk 1]
```diff
@@ -0,0 +1,4 @@
+name = Custom Theme tasks
+description = "Custom module for EPAM's drupal course. Module: Theme"
+package = EPAM Drupal Training
+core = 7.x
```
Comment (dprudnikov) [inline docroot/sites/all/modules/custom/theme_tasks/theme_tasks.info]: Add empty line

File: docroot/sites/all/modules/custom/theme_tasks/theme_tasks.pages.inc [hunk 2]
```diff
@@ -0,0 +1,128 @@
+<?php
+
+/**
+ * @file
+ * Module with theme tasks.
```
Comment (dprudnikov) [inline docroot/sites/all/modules/custom/theme_tasks/theme_tasks.pages.inc]: Write user-friendly comment

File: docroot/sites/all/modules/custom/theme_tasks/theme_tasks.pages.inc [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+ */
+
+/**
+ * Landing page content.
+ *
+ * @return string
+ *   string with html.
```
Comment (dprudnikov) [inline docroot/sites/all/modules/custom/theme_tasks/theme_tasks.pages.inc]: Fix code style

File: docroot/sites/all/modules/custom/theme_tasks/theme_tasks.module [hunk 4]
```diff
@@ -0,0 +1,70 @@
+<?php
+
+/**
+ * @file
+ * Module with theme tasks.
+ */
+
+/**
```
Comment (dprudnikov) [inline docroot/sites/all/modules/custom/theme_tasks/theme_tasks.module]: Place hook_permission below hook_menu

File: docroot/sites/all/modules/custom/theme_tasks/theme_tasks.info (hunk 1 above)
Comment (dkorhov) [inline docroot/sites/all/modules/custom/theme_tasks/theme_tasks.info]: Fixed in a6153e1

File: docroot/sites/all/modules/custom/theme_tasks/theme_tasks.module [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+ * Implements hook_theme().
+ */
+function theme_tasks_theme() {
+  return array(
+    'theme_tasks_protected_output' => array(
+      'variables' => array('string' => t('Custom access inverted for rendered content')),
+      'file' => 'theme_tasks.pages.inc',
+      'template' => 'theme_tasks_protected',
```
Comment (dkorhov) [inline docroot/sites/all/modules/custom/theme_tasks/theme_tasks.module]: Moved in a6153e1

File: docroot/sites/all/modules/custom/theme_tasks/theme_tasks.pages.inc (hunk 2 above)
Comment (dkorhov) [inline docroot/sites/all/modules/custom/theme_tasks/theme_tasks.pages.inc]: Fixed in a6153e1

**Pull request merged**